acquire/utils.py

```diff
@@ -124,8 +124,17 @@ def create_argument_parser(profiles: dict, volatile: dict, modules: dict) -> arg
     parser.add_argument("-p", "--profile", choices=profiles.keys(), help="collection profile")
     parser.add_argument("--volatile-profile", choices=volatile.keys(), help="volatile profile")
 
-    parser.add_argument("-f", "--file", action="append", help="acquire file")
-    parser.add_argument("-d", "--directory", action="append", help="acquire directory recursively")
+    # Keep `--file` and `--dir` (-f, and -d) temporarily
+    parser.add_argument(
+        "-f",
+        "-d",
+        "--file",
+        "--dir",
+        "--path",
```

Comment:
The only point I'm torn on is if it's worth keeping the arguments around, especially for their shorthand versions. Perhaps we can drop that question with the analysts before removing them.